database/migrations/2019_05_03_000001_create_customers_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCustomersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('customers', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('billable_id');
+            $table->string('billable_type');
+            $table->string('paddle_id')->unique();
+            $table->string('paddle_email')->unique();
```

Comment:
Wouldn't it make sense to make both of the columns nullable?

In it's current form it's not possible to create a customer without paddle_id/email. For example in the case of setting the `trial_ends_at` during the default Laravel registration process.

Reply:
That's an excellent point. I'll update that 👍